newrelic/packages/isort/stdlibs/py312.py

```diff
@@ -0,0 +1,220 @@
+"""
+File contains the standard library of Python 3.12.
+
+DO NOT EDIT. If the standard library changes, a new list should be created
+using the mkstdlibs.py script.
+"""
+
```

Comment:
Is this file necessary at all? Based on the below code I don't think we ever reference it.

```python
    elif python_version >= (3, 10):
        stdlibs = sys.stdlib_module_names
```

Reply:
It really doesn't require python 3.10 or 3.11 either, but those were probably just copied in since they were there. I'm unsure if there's any reason to update these.

Reply:
Ah true. It looks like there were only changes to 3.11 and 3.12 here. Given that both those aren't used, I guess we can just skip upgrading isort for now.